### ISSUE TITLE
File type associations not installed

### ISSUE THREAD

**Issue by mueslo** [jupyterlab/jupyterlab-desktop#355]
## Description

The `README.md` claims that click to open should work for the GUI file explorer:

>## Launching JupyterLab Desktop
>
>JupyterLab Desktop can be launched from the GUI of your operating system by clicking the application's icon or by using jlab command from the command line. Double clicking .ipynb files is also supported and it will launch JupyterLab Desktop and load the notebook file.

## Reproduce

1. Install JupyterLab Desktop
2. Navigate with `explorer.exe` to desired `*.ipynb` notebook file
3. Find that no file association for opening in JupyterLab Desktop is installed so double clicking or right clicking to open in JupyterLab Desktop does not work.

## Expected behavior

Work as advertised

## Context

- Operating System and version: Windows 10
- JupyterLab-Desktop version: 3.2.4-3
- Installer: .exe

**Comment by mbektas** [comment]
@mueslo It is required to make the file type association once on your system. JupyterLab Desktop will be listed as one of the supported applications which can open .ipynb files. You need to select it and make it the default application for the file type. Next time you double click .ipynb files, you won't be asked to choose which application to use.

**Comment by mueslo** [comment]
No, it was not. I had to manually navigate to C:\JupyterLab\jlab.exe and asociate it.
